index.js

```diff
@@ -137,6 +137,8 @@ function SendStream(req, path, options) {
   if (!this._root && opts.from) {
     this.from(opts.from)
   }
+
+  mime.default_type = opts.defaultType || null;
```

Comment:
You're right, thanks for pointing me out this issue, I'll update this

Does it means that currently send may be affected when mime.default_type is set in another module (and possibly breaks if it is null)?